docs/baseplate/core.rst

```diff
@@ -58,6 +58,38 @@ integrations <integration/index>`.
 .. autoclass:: TraceInfo
    :members: from_upstream
 
+Authentication
+--------------
+
+If the application is dealing with authentication either directly from the
+authentication service or from upstream services, there are some support interfaces
+available for easily working with passing context down or picking up passed down
+contexts.
+
+When receiving an initial authentication token from the authentication service, the
+application should properly store the token in an
+:py:class:`~baseplate.core.AuthenticationContext` object attached to the current
+context::
+
+    authentication_token = retrieve_token_from_authentication_service()
+    authentication_context = AuthenticationContext(authentication_token, secrets_store)
+    authentcation_context.attach_context(context)
```

Comment:
The problem is that I have a "magical" method for showing token retrieval and undefined `secrets_store` and `context` variables used... :(